cmd/document-crds.go

```diff
@@ -20,7 +20,9 @@ func newDocumentCRDsCommand(log logr.Logger) (*cobra.Command, error) {
 		Use:   "crds",
 		Short: "Generate CRD documentation from a package",
 		Long:  "Generate CRD documentation from a package.",
-		RunE: func(_ *cobra.Command, args []string) error {
+		RunE: func(c *cobra.Command, args []string) error {
+			options.cmd = c
```

Comment:
The options struct is supposed to capture the chosen options independently of cobra. Don't tightly couple things like this, it makes things harder to maintin.